stories/button.stories.js

```diff
@@ -40,6 +41,50 @@ storiesOf('Core|Button', module)
       </p>
     </React.Fragment>
   ))
+  .add('appearance:default-with-icon', () => (
+    <React.Fragment>
+      <p>
+        <Button
+          icon={ArrowRightRegular}
+          iconPosition="center"
+          intent="info"
+        ></Button>
+      </p>
+      <p>
+        <Button icon={ArrowRightRegular} iconPosition="left" disabled>
+          Hulk smash!
+        </Button>
+      </p>
+      <p>
+        <Button icon={ArrowRightRegular}>Hulk smash!</Button>
+      </p>
+      <p>
+        <Button icon={ArrowRightRegular} intent="info">
+          Hulk smash!
+        </Button>
+      </p>
+      <p>
+        <Button icon={ArrowRightRegular} intent="success">
+          Hulk smash!
+        </Button>
+      </p>
+      <p>
+        <Button icon={ArrowRightRegular} intent="warning">
+          Hulk smash!
+        </Button>
+      </p>
+      <p>
+        <Button icon={CalendarRegular} intent="danger">
+          Hulk smash!
+        </Button>
+      </p>
+      <p>
+        <Button icon={ArrowRightRegular} full>
```

Comment:
`full` Buttons somehow break margins on the Safari browser:
![Zrzut ekranu 2019-10-29 o 07 56 13](https://user-images.githubusercontent.com/8265332/67744376-9a3c0400-fa21-11e9-93ad-1ff92305ec89.png)

Reply:
Cause found (https://github.com/google/material-design-lite/issues/4008), resolved with https://github.com/eyedea-io/smashing-ui/pull/59/commits/b6621edc0b1f1095d3a00791c14617a088e60817